torch/utils/data/__init__.pyi

```diff
@@ -0,0 +1,39 @@
+# These imports are necessary for functional DataPipe code completion
```

Comment:
I am super confused by this file. Why this is necessary?

Reply:
I think you are right and this is actually not needed. I will remove that file.